java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java

```diff
@@ -0,0 +1,29 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Supports commands and transactions for the "Bitmap Commands" group for standalone and cluster
+ * clients.
+ *
+ * @see <a href="https://redis.io/docs/latest/commands/?group=bitmap">Bitmap Commands</a>
+ */
+public interface BitmapBaseCommands {
+    /**
+     * Returns the bit value at <code>offset</code> in the string value stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/getbit/">redis.io</a> for details.
+     * @param key The key for the string to get the bit at <code>offset</code> of.
+     * @param offset The index of the bit to return.
+     * @return The bit at offset of the string. Returns zero if the key is missing as it is treated as
+     *     an empty string. Returns zero if the positive <code>offset</code> exceeds the length of the string as it
+     *     is assumed to be padded zeroes.
+     * @example
+     *     <pre>{@code
+     * Long payload = client.getbit("myKey1", 1).get();
+     * assert payload == 1L; // The second bit for string stored at "myKey1" is set to 1.
+     * }</pre>
+     */
+    CompletableFuture<Long> getbit(String key, long offset);
```

Comment:
consider something smaller than Long for the return type.

Reply:
I thought we only had Long and Double types for all number types. Would introducing Integer/Byte/Short cause any issues?

Reply:
lets stick with long until we have a consensus

Reply:
As per the discussion offline, we are going to use long. This can be switched in the future if a boolean or byte is preferred.